integration_test/test_videoroom/test/integration/simulcast_test.exs

```diff
@@ -77,7 +77,7 @@ defmodule TestVideoroom.Integration.SimulcastTest do
     }
 
     sender_actions = [
-      {:get_stats, @simulcast_inbound_stats, @stats_number, @stats_interval, tag: :after_warmup},
```

Comment:
Was it unused or was it causing errors? 😰

Reply:
We've determined it to be a typographical error related to the Copy-Paste Algorithm, most likely :smile:

Funny (or depressing, depending how you look at it) thing is, it didn't make the tests fail...

Reply:
So we don't use it anywhere? If yes, we should use it somewhere or remove it, IMO.

Reply:
No, it is used, the stats are getting compared when running the test. Here's what was happening:

1. The author's intention was to always gather `simulcast-outbound-stats` from one browser (the sender) and always gather `simulcast-inbound-stats` from the other one (the receiver)
2. In `sender_actions`, the first `:get_stats` action was erroneously gathering the wrong kind of stats (`simulcast-inbound-stats`). The rest of actions were fine
3. When comparing the results, in the function `assert_sender_receiver_stats`, the stats were passed as positional arguments -- `sender_stats_samples` and `receiver_stats_samples`. The logic inside this function expected these arguments to be `simulcast-outbound-stats` and `simulcast-inbound-stats`, respectively
4. But, as it turns out, all of the assertions passed even when `sender_stats_samples` were of the wrong kind, `simulcast-inbound-stats` ¯\_(ツ)_/¯

Then I made the following changes:
1. Changed the structure of each sample from `simulcast-inbound-stats` to a list, to be able to get stats from more than one other browser
2. Changed the `assert_sender_receiver_stats` function to flatten this new list (the one passed as `receiver_stats_samples`) before proceeding, as there was always only one other browser used in simulcast tests
3. Didn't change anything with regards to the stats passed as `sender_stats_samples`, as I expected them to always be `simulcast-outbound-stats`, which haven't changed their structure

All of the simulcast tests started raising errors, because at one time `simulcast-inbound-stats` (with the changed structure) were passed as `sender_stats_samples` (for which the function was expecting the old structure) -- and thus, the mistake hidden in plain sight was uncovered at last...